framework/src/ble_manager/Kconfig

```diff
@@ -0,0 +1,18 @@
+#
+# For a description of the syntax of this configuration file,
+# see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
+#
+
+config BLE_MANAGER
+    bool "Enable BLE Easy Manager"
```

Comment:
What is the name of this module? BLE manager or BLE easy manager?
To me, the former is better.

Reply:
Ok, I will remove it